src/Microsoft.AspNetCore.BuildTools.ApiCheck/build/Microsoft.AspNetCore.BuildTools.ApiCheck.props

```diff
@@ -0,0 +1,15 @@
+﻿<!--
+  WARNING: These targets are intended for building Microsoft's ASP.NET Core repos and are not intended
+  for use outside of Microsoft.
+-->
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)Common.props"/>
+
+  <PropertyGroup>
+    <_ApiCheckTaskAssembly>$(MSBuildThisFileDirectory)..\tools\netstandard1.6\Microsoft.AspNetCore.BuildTools.ApiCheck.Task.dll</_ApiCheckTaskAssembly>
```

Comment:
Intentionally not part of non-Core MSBuild. But, if I left out the condition, I'll add it.